core/domain/html_cleaner.py

```diff
@@ -149,7 +150,7 @@ def get_image_filenames_from_html_strings(html_strings):
     return list(set(filenames))
 
 
-def get_rte_components(html_string):
+def get_rte_components(html_string: str) -> List[Dict[str, Any]]:
```

Comment:
Why is it `Dict[str, Any]` ?

the object has only two keys right? `id` and `customization_args`

`customization_args` is again a dict, I guess.

Reply:
Yeah true. I have added TypedDict in my last [commit](https://github.com/oppia/oppia/pull/15435/commits/f47a2010a75aafe65376ad399e6773bc5098d250), PTAL. Also, can you please reply to [this](https://github.com/oppia/oppia/pull/15435/commits/f47a2010a75aafe65376ad399e6773bc5098d250#r870637597) one? I hope it will resolve this conversation as well.